src/ui/qgsdiagrampropertiesbase.ui

```diff
@@ -2376,70 +2176,6 @@
     </hint>
    </hints>
   </connection>
-  <connection>
-   <sender>mFixedSizeRadio</sender>
-   <signal>toggled(bool)</signal>
-   <receiver>mDiagramSizeSpinBox</receiver>
-   <slot>setEnabled(bool)</slot>
-   <hints>
-    <hint type="sourcelabel">
-     <x>111</x>
-     <y>153</y>
-    </hint>
-    <hint type="destinationlabel">
-     <x>235</x>
-     <y>154</y>
-    </hint>
-   </hints>
-  </connection>
-  <connection>
-   <sender>mAttributeBasedScalingRadio</sender>
```

Comment:
@roya0045 , looking at this today -- did you mean to the connection stuff here?

Reply:
OK, the missing connection was definitively a regression. Please, make sure you test your changes before opening PRs. 

See the non functional UI due to missing connections:
![Peek 2022-03-05 13-31](https://user-images.githubusercontent.com/1728657/156871530-9fbc1a0f-b87e-4fe4-96df-2688cb9f89a4.gif)

Reply:
The connection removal was related to another branch and I forgot to clean this up when separating the two. Thanks for pointing it out!